sim/src/config.h

```diff
@@ -37,6 +37,7 @@ namespace crimson {
       double client_reservation;
       double client_limit;
       double client_weight;
+      uint64_t client_req_cost;
```

Comment:
can stuff in sim/ use the Cost typedef?

Reply:
I ended up doing an odd work-around here. The simulator defines its own Cost and then I assert upon start-up that the sizeofs of the two are the same. I played a bit with SFINAE but couldn't get it to work and didn't want to spend more time on it. Your thoughts?